### ISSUE TITLE
Ability to create a web-app for the installed browser

### ISSUE THREAD

**Issue by visvital** [artemanufrij/webpin#75]
As long as, webpin only allows to run websites and apps which are properly supported by WebKit, as a user, I would like to be able to create the web apps for my list of installed browsers using the webpin.

That would allow me to control and customize the web app I'm creating, as well as to avoid the problem with their improper display.

**Comment by caiohofbauer** [comment]
Yes, I would like that, since the web apps I use doesn't work on WebPin. I would like to run them on Firefox.
